assets/stylesheets/units/_forms.scss

```diff
@@ -0,0 +1,17 @@
+textarea {
+  @include core-font(20);
+
+  border: ($baseline-grid-unit / 2) solid $grey-2;
+  display: block; // Remove margin under textarea in Chrome and FF
+  margin: ($baseline-grid-unit * 4) 0;
```

Comment:
Should we be keeping spacing consistent and using the `element-spacing` mixin which also accounts for screen sizes?

So `16px` on smaller devices and `32px` on larger devices.